BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs

```diff
@@ -36,6 +38,31 @@ public void HandlePlayerClicked(Player player)
         _navigationManager.NavigateTo("/PlayerDashboard/" + player.Id);
     }
 
+    public string GetTrueSkillChange(Player player, string? matchId)
+    {
+        var trueSkillChange = "-";
+        if (Matches.Any())
+        {
+            var matchIndex = Matches.FindIndex(x => x.Id == matchId);
+            for (int i = matchIndex + 1; i <= Matches.Count; i++)
+            {
+                var match = Matches[i];
+                if (!match.Players.Any(x => x.Id == player.Id))
+                {
+                    continue;
+                }
+
+                var prevMatchPlayer = match.Players.Single(x => x.Id == player.Id);
+
+                trueSkillChange =
+                    ((player.TrueSkillRating!.Mean - prevMatchPlayer!.TrueSkillRating!.Mean) * 100).ToString("+#;-#;0");
```

Comment:
I have also added some logic to convert true skill. I know that there are already 2 different methods for transforming win rate to percentages... I think we need to do something about that

Reply:
You don't have to do anything, about this, just letting you know :D

Reply:
Yea, I noticed that in your PR - thanks! :)